.github/ISSUE_TEMPLATE/documentation_request.yml

```diff
@@ -0,0 +1,64 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Documentation Request
+description: Request updates or additions to cuDF's documentation
+title: "[DOC]: "
+labels: ["doc"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuDF and improve our documentation!
+        - By submitting this issue, you agree to follow our [Code of Conduct](https://docs.rapids.ai/resources/conduct/)
+        - You also confirm that you have searched the [open documentation issues](https://github.com/rapidsai/cuDF/issues?q=is%3Aopen+is%3Aissue+label%3Adoc) and have found no duplicates for this request
+
+  - type: dropdown
+    id: criticality
+    attributes:
+      label: How would you describe the priority of this documentation request
+      options:
+        - Critical (currently preventing usage)
+        - Medium
+        - Low (would be nice)
+    validations:
+      required: true
+
+  - type: dropdown
+    id: new_or_correction
+    attributes:
+      label: Is this for new documentation, or an update to existing docs?
```

Comment:
This is fuzzy to me -- and thus potentially a source of friction for issue filers. If a function is undocumented, that's probably "new." But is an undocumented keyword for an existing function argument supposed to be "new" or "update"?

Reply:
Does it matter whether the request is for "new" or "update" from the point of view of the reporter? It might matter for _us_, but that is an internal "implementation" detail.

Reply:
I think it matters to both parties - it frames the issue with minimal effort (2 mouse clicks). It also lets the user know what we expect of them since the expected responses in the downstream text boxes are conditional (if only the form could be conditional) based on if it's new or an update. (or `incorrect` vs `needed`).

Reply:
I would like to remove this dropdown. As a filer, I don't think it adds enough value for the 2 click friction.